src/ecs_cluster/ecs_client.py

```diff
@@ -324,6 +325,7 @@ def start_task(self, cluster_name, task_definition):
 
     # pylint: disable=too-many-locals
     def docker_stats(self, cluster_name, ssh_keydir, user):
+        # pylint: disable=unnecessary-comprehension
         arns = [x for x in self.ecs_client.list_container_instances(
```

Comment:
should just be `arns = self.ecs_client.list_container_instances(cluster=cluster_name)["containerInstanceArns"])`, no?

Reply:
ah yeah! updated it